### ISSUE TITLE
[1.3-rc.3] import-scanner - inconsistent lazy loading package files

### ISSUE THREAD

**Issue by ecwyne** [meteor/meteor#6527]
Build fails when packages use `api.addFiles` with a relative path (eg. `./local-package.js`).
Specifically the [import-scanner](https://github.com/meteor/meteor/blob/release-1.3/tools/isobuild/import-scanner.js#L142) shows that `oldFile.lazy` !== `newFile.lazy`

### Example
```javascript
// package.js
Package.onUse(function(api) {
  api.versionsFrom('1.2.1');
  api.use('ecmascript');
  api.addFiles('./local-package.js'); // file added with relative path
});
```

Reproduction repository to follow...
cc @benjamn

**Comment by ecwyne** [comment]
Reproduction: https://github.com/ecwyne/meteor-issue-6527
Most important line is https://github.com/ecwyne/meteor-issue-6527/blob/master/packages/local-package/package.js#L10

This only occurs when the package uses the `ecmascript` package

### To Reproduce
1. `$ git clone https://github.com/ecwyne/meteor-issue-6527`
2. `$ cd meteor-issue-6257`
3. `$ meteor`

### Result
```shell
<runJavaScript-54>:168
      throw error;
            ^
AssertionError: false === true
    at ImportScanner._combineFiles (/tools/isobuild/import-scanner.js:142:12)
    at /tools/isobuild/import-scanner.js:125:14
    at Array.forEach (native)
    at ImportScanner.addInputFiles (/tools/isobuild/import-scanner.js:103:11)
    at /tools/isobuild/compiler-plugin.js:725:15
    at Array.forEach (native)
    at Function.computeJsOutputFilesMap (/tools/isobuild/compiler-plugin.js:695:19)
    at ClientTarget._emitResources (/tools/isobuild/bundler.js:874:8)
    at /tools/isobuild/bundler.js:641:12
    at /tools/utils/buildmessage.js:359:18
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:352:34
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:350:23
    at [object Object].withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.enterJob (/tools/utils/buildmessage.js:324:26)
    at ClientTarget.make (/tools/isobuild/bundler.js:632:18)
    at /tools/isobuild/bundler.js:2397:14
    at /tools/isobuild/bundler.js:2487:20
    at Array.forEach (native)
```

### Expected
Normal starting of meteor application

**Comment by benjamn** [comment]
Thanks for the reproduction. Fix coming soon!

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.4`](https://github.com/meteor/meteor/issues/6266#issuecomment-198997482).

**Comment by MHerszak** [comment]
Hi @benjamn, I still have this issue with 1.3. Any idea?
